### ISSUE TITLE
Error: Please create `.dploy_config.json` file in HOME_DIR

### ISSUE THREAD

**Issue by ingro** [voronianski/node-tiny-dploy#1]
Hello, I wanted to try your cool script but I got this error when I type `dploy setup`, here's the full error log:

```
Error: Please create `.dploy_config.json` file in HOME_DIR
Error: Or run `dploy setup` to create first instance
```

I'm running this on Ubuntu 14.04, if you need something else to better identify the problem let me know, keep up the good work!

**Comment by voronianski** [comment]
@ingro thanks, update please, fixed in `0.2.18`.

**Comment by ingro** [comment]
Thanks @voronianski , I will check asap!
